.github/workflows/flp-dependencies-pr.yml

```diff
@@ -0,0 +1,33 @@
+name: PR update Free Law Project dependencies
+
+on:
+  workflow_dispatch:
+
+jobs:
+  update-flp-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+
+      - name: Install Poetry
+        run: |
+          curl -sSL https://install.python-poetry.org | python3 -
+
+      - name: Update FLP dependencies
+        run: |
+          poetry update juriscraper reporters-db courts-db eyecite
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.TOKEN }}
+          commit-message: Update freelawproject dependencies
+          title: Update freelawproject dependencies
+          body: This PR updates the juriscraper package.
```

Comment:
Can you fix this?

Reply:
I think I need your knowledge about what/generate the tokens to use

Reply:
No, I mean the `body` key, which is incorrect. I think the secrets might be fine.

Reply:
great.  ive updated it